composer.json

```diff
@@ -28,7 +28,7 @@
     "ext-pdo": "*",
     "contao-community-alliance/meta-palettes": "^2.0",
     "contao/core-bundle": "^4.9",
-    "doctrine/cache": "^1.0",
+    "doctrine/cache": "^2.1",
```

Comment:
If you remove the dependency of doctrine/cache, why is it still in the `composer.json`?

```suggestion
```

Reply:
Furthermore, if you depend on `symfony/cache`, it has to be added to the requirement as well